src/gui/accountsettings.cpp

```diff
@@ -845,19 +845,28 @@ void AccountSettings::slotAccountStateChanged()
             serverWithUser = tr("%1 as <i>%2</i>").arg(server, Utility::escape(user));
         }
 
-        if (state == AccountState::Connected) {
+        switch(state)
```

Comment:
(nitpick:  we follow the Qt style which put the brace on the same line  (here and after the case)